airgun/views/contenthost.py

```diff
@@ -106,7 +106,9 @@ class ContentHostsView(BaseLoggedInView, SearchableViewMixin):
             'Installable Updates': InstallableUpdatesCellView(),
         },
     )
-    pagination = Pagination()
```

Comment:
Any way we can fix the pagination for all the airgun entitites? 
- By using the unique id for `total_items` and `pages` through pagination function directly ?
- If the unique ID is not present, shouldnt we file a bug to the DEVs that the ID is changing just for Content Host entity ?

The changing ID has been consistent issue in Satellite UI.